### PR TITLE
docs: FormsModule and ReactiveForms modules....

### PR DESCRIPTION
**Pull request by raja1203**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by aikidave** [comment]
Hi @raja1203! Can you provide some more information about what you're trying to accomplish with this PR? Thanks!

**Comment by raja1203** [comment]
Hi,

Good day.

I am refreshing my knowledge on Angular. As a part of this exercise, I have
started building Shopping cart App. While creating forms, there is no where
it is mentioned that you need to include FormsModule and Reactive Forms
module in app.module.ts.

This is particularly important especially for those people who started
learning it for the first time. Due to my earlier knowledge , I have
included FormsModule. but still it didn't work . After checking in
Stackoverflow , I have included ReactiveForms Module also. Then it worked.

What I am trying to achieve is correction in content which will increase
completeness of the angular tutorial.
If you need any more clarification, please let me know.


Thanks and Regards

Rajasekhar

File: aio/content/start/start-forms.md [hunk 1]
```diff
@@ -8,6 +8,28 @@ This section walks you through adding a form-based checkout feature to collect u
 
 Forms in Angular build upon the standard HTML forms to help you create custom form controls and easy validation experiences. There are two parts to an Angular Reactive form: the objects that live in the component to store and manage the form, and the visualization of the form that lives in the template.
 
+First, in app.module.ts
```
Comment (petebacondarwin) [inline aio/content/start/start-forms.md:11]:
```suggestion
First, in `app.module.ts`
```

File: aio/content/start/start-forms.md [hunk 2]
```diff
@@ -8,6 +8,28 @@ This section walks you through adding a form-based checkout feature to collect u
 
 Forms in Angular build upon the standard HTML forms to help you create custom form controls and easy validation experiences. There are two parts to an Angular Reactive form: the objects that live in the component to store and manage the form, and the visualization of the form that lives in the template.
 
+First, in app.module.ts 
+import FormsModule and ReactiveForms Module. Next register them under @NgModule. Code snippet.
```
Comment (petebacondarwin) [inline aio/content/start/start-forms.md:12]:
```suggestion
import `FormsModule` and `ReactiveFormsModule`. Next register them under `@NgModule`. ```

Also the code snippet below should actually reference code in a real example. Something similar to:

```
<code-example header="src/app/app.module.ts" path="getting-started/src/app/app.module.ts" region="forms-module-imports"></code-example>
```

But that also requires changes to the example code itself to include the `import {FormsModule} from '@angular/forms';` line and also comments to define the region. So `aio/content/examples/getting-started/src/app/app.module.ts` would need to look like:

```
// #docregion forms-module-imports
import { ReactiveFormsModule, FormsModule } from '@angular/forms';
// #enddocregion forms-module-imports
```

**Review by petebacondarwin — CHANGES_REQUESTED** [review]
Thanks for this PR. Before it can be merged we need the CLA to be signed and also for you to address the following comments.

**Comment by jelbourn** [comment]
Closing this since the CLA was never signed.

**Pull request closed without merging**